lib/edoc/src/edoc_extract.erl

```diff
@@ -32,15 +32,10 @@
 %% %% @headerfile "edoc.hrl" (disabled until it can be made private)
 -include("edoc.hrl").
 
-%% @type filename() = file:filename().
-%% @type proplist() = proplists:property().
-%% @type syntaxTree() = erl_syntax:syntaxTree().
-
-%% @spec source(File::filename(), Env::edoc_env(), Options::proplist())
-%%             -> {ModuleName, edoc:edoc_module()}
-%%    ModuleName = atom()
-%%    proplist() = [term()]
-%%
+-type filename() :: file:filename().
+-type proplist() :: proplists:property().
+-type syntaxTree() :: erl_syntax:syntaxTree().
+
```

Comment:
Please add local <code>edoc_env</code, <code>edoc_module</code> and <code>comment</code> types to use instead of the ones with module prefix. This will make the documentation more readable.